README.md

```diff
@@ -50,3 +50,41 @@ implementers SHOULD:
 
 - ignore additional, unknown fields present in a JSON object:
   This allows the addition of new (optional) fields to a version without breaking implementations.
+
+## Client implementations
+
+While the standards are relatively simple implementation-wise
+as it is just a JSON
+wrapped in gzip compression and base64 encoding with a header,
+it can be helpful to have a client library
+which provides certain types and utilities
+for easier creation and handling of medication plans.
```

Comment:
I don't know if this sentence adds anything. I would also refrain from calling things "simple" and using "just"

Reply:
Good call, I tried to add this as an introduction sentence but it does not any useful information, so let's just remove it 👍